test/models/team_test.rb

```diff
@@ -135,4 +135,15 @@ def setup
     assert_equal challenge.point_value, team.score
   end
 
+  test 'cannot create team if require_team is true but no team is given' do
+    @game = create(:active_game, location_required: true)
+    assert_raises(ActiveRecord::RecordInvalid) {
+      team = create(:team)
```

Comment:
You could assert here that the error is appropriate and that the `team.errors` contains the "location is required" error.